### ISSUE TITLE
Biceps tendonitis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10826]
**Preferred term label:**
Biceps tendonitis

**Synonyms**
Also spelled tendinitis

**Definition (free text, please give PubMed ID)**
MRI showing increased T2 signal around the biceps tendon or ultrasound showing thickening of the biceps tendon

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Biceps tendonitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001146
- label: Biceps tendonitis
- parents: [Tendonitis (HP:0025230)](https://hpo.jax.org/app/browse/term/HP:0025230)
- pmid: PMID:19725488
- def: Biceps tendinitis is inflammation of the tendon around the long head of the biceps muscle.
- comment: MRI may show increased T2 signal around the biceps tendon or ultrasound showing thickening of the biceps tendon.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10826
- synonyms: Biceps tendinitis(NONE)
